test/evergreen.yml

```diff
@@ -3370,6 +3370,78 @@ tasks:
         vars:
           perf-test-name: modify-force-update-large-record-btree.wtperf
 
+    ###############################
+    # Performance Tests for oplog #
+    ###############################
+
+  - name: perf-test-mongodb-oplog
+    tags: ["oplog-perf"]
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "run-perf-test"
+        vars:
+          perf-test-name: mongodb-oplog.wtperf
+          maxruns: 2
+          wtarg: -ops ['"load", "insert", "truncate", "database_size"']
+      - func: "upload-perf-test-stats"
+        vars:
+          perf-test-name: mongodb-oplog.wtperf
+
+  - name: perf-test-mongodb-small-oplog
+    tags: ["oplog-perf"]
+    depends_on:
+      - name: compile
+    commands:
+      - command: timeout.update
+        params:
+          exec_timeout_secs: 10800
```

Comment:
This and `perf-test-mongodb-large-oplog` have `run_time=3600` in their .wtperf files. Is there a reason we need the additional timeout check?

Reply:
The timeout setting was required for the case where we run the tests 3 times (as one run takes an hour). Now that we are changing the frequency to one. I will get rid of the timeout settings.
Updated in [d9678b7](https://github.com/wiredtiger/wiredtiger/pull/8366/commits/d9678b71fa50399c50aee748f81e8cdfd8727d06)